tools/alive_parser.cpp

```diff
@@ -220,9 +220,38 @@ static void parse_comma() {
 }
 
 
+static unordered_map<unsigned, unique_ptr<AggregateType>> overflow_aggregate_types;
 static vector<unique_ptr<SymbolicType>> sym_types;
 static unsigned sym_num;
 
+static Type& get_overflow_type(unsigned bitwidth) {
+  if (overflow_aggregate_types.find(bitwidth) != overflow_aggregate_types.end()) {
+    return *overflow_aggregate_types[bitwidth];
```

Comment:
this code does 3 map lookups, while it can be change to do only emplace (that is patched if type not there).